deploy/docker-compose.yml

```diff
@@ -12,9 +12,8 @@ services:
       - "8081:8081"
     stop_grace_period: 0s
     environment:
```

Comment:
What value does docker-compose.yml add over directly run the pipeline-manager using docker run?

Reply:
Would be way easier to keep redpanda, grafana and prometheus in this one as before and a much more incremental change. The other services indeed can be carved off into their own compose files if you'd like.

Reply:
There is no added value except that with docker-compose you can combine it more easily with other docker-compose -- it's like a convenience template we provide that people can use in their own docker compose files, or to chain with their own

Reply:
I've brought back Redpanda, Grafana and Prometheus in the main docker-compose.yml